module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,15 @@
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& fList) {
+    std::deque<std::string> deq;
+    
+    fList.sort([](const std::string& lhs, const std::string& rhs) {
+        return lhs.size() == rhs.size() ? lhs < rhs : lhs.size() < rhs.size();
+    });
+
+    for (const auto& el : fList) {
+        deq.push_back(el);
+    }
```

Comment:
Declare std::deque at end and initialize it with iterators `std::deque<std::string> result {fList.begin(), fList.end()}`

Reply:
Pushing each value is not efficient. Try in the way suggested by @nauka-programowania-MA / @kuchichan